### ISSUE TITLE
Stackage nightly test failure

### ISSUE THREAD

**Issue by lwm** [jeremyjh/higher-leveldb#7]
For `higher-leveldb-0.4.0.0`, when running the tests, I saw:

```
/usr/bin/ld.gold: warning: cannot find entry symbol 'nable-new-dtags'
```

As this is a new package, we normally disable it until we get a clean run.

**Comment by lwm** [comment]
Oh wait, I can just disable the test suite, will fix!

**Comment by jeremyjh** [comment]
@lwm Thanks, and sorry about that! I reproduced the problem and uploaded a fix for it to Hackage as 0.4.0.1.  There was an old compiler flag causing the issue in the test executable. Can we re-enable tests for it?

**Comment by lwm** [comment]
Re-enabled!
